internal/bminventory/inventory.go

```diff
@@ -273,8 +275,7 @@ func (b *bareMetalInventory) UpdateDiscoveryIgnition(ctx context.Context, params
 func (b *bareMetalInventory) UpdateDiscoveryIgnitionInternal(ctx context.Context, params installer.UpdateDiscoveryIgnitionParams) error {
 	log := logutil.FromContext(ctx, b.log)
 
-	//[TODO] - change the code to use InfraEnv once we move the code and test to use InfraEnv CRUD
-	c, err := b.getCluster(ctx, params.ClusterID.String())
+	infraEnv, err := common.GetInfraEnvFromDB(b.db, params.ClusterID)
```

Comment:
This function is used for v1 as well, i tihnk that this change will break it. 
@yevgeny-shnaidman what is the ticket for v2 update discovery ignition?

Reply:
MGMT-7520

Reply:
ignition-override  currently is part of the UpdateInfraEnv paramters